src/caret_analyze/value_objects/message_context.py

```diff
@@ -124,6 +124,14 @@ def subscription_topic_name(self) -> Optional[str]:
             return None
         return self._sub.topic_name
 
+    @property
+    def publisher_construction_order(self) -> Optional[int]:
+        return None
+
+    @property
+    def subscription_construction_order(self) -> Optional[int]:
+        return None
```

Comment:
Need to return construction_order of subscription.

```suggestion
        if self._sub is None:
            return None
        return self._sub.construction_order
```

Reply:
I corresponded.